src/slskd/Options.cs

```diff
@@ -694,6 +694,15 @@ public class SoulseekOptions
             [RequiresReconnect]
             public string Password { get; init; } = null;
 
+            /// <summary>
+            ///     Gets the description of the Soulseek user.
+            /// </summary>
+            [Argument(default, "slsk-description")]
+            [EnvironmentVariable("SLSK_DESCRIPTION")]
+            [Description("user description for the Soulseek network")]
+            [RequiresReconnect]
```

Comment:
Since this is resolved dynamically and the `Options` is a shortcut for `OptionsMonitor.CurrentValue`, changes are reloaded on the fly and won't require a reconnect to update.

Reply:
Good catch! I'll update this soon.